src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const generalCSQuiz = [
+  {
+    Question:
+      "What is a subnet?",
+    Answer: "A smaller network within a larger one",
+    Distractor1: "A backup system for when a network crashes",
+    Distractor2: "A substitution method used in website domains",
+    Distractor3: "A cybersecurity access point",
+    Explanation:
+      "A subnet, or subnetwork, is a smaller network within a larger one.",
+    Link: "https://www.freecodecamp.org/news/subnet-definition/",
```

Comment:
We already have this question about subnets in our information technology quiz. 
Please come up with a new question.

I would suggest finding a question around computer science algorithms since we are low on those.
https://www.freecodecamp.org/news/search?query=algorithms

Reply:
Great catch! I didn't think to check the IT quiz but I think it makes sense for it to be over there.Thanks for the redirection. Will submit a change shortly.